.github/workflows/github-ci.yml

```diff
@@ -27,7 +27,11 @@ env:
   IPCL_DIR: ${GITHUB_WORKSPACE}/ipcl_install
   IPCL_HINT_DIR: >
       -DIPCL_HINT_DIR=${GITHUB_WORKSPACE}/ipcl_install/lib/cmake/ipcl-${IPCL_VER}
-
+      
+permissions:
+  contents: read
+  
+jobs:
```

Comment:
It looks like there's a duplicate `jobs` here. (line 34 and 35)

Reply:
Sorry about that. I fixed the error.